code/src/components/NameForm.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export const Name = ({ name, setName }) => {
+  const handleNameChange = (event) => {
+    setName(event.target.value);
+  }
+  return (
+    <div className="introduction">
+      <h1>Website feedback form</h1>
+      <h3>Hi, my name is Emilia. Who are you?</h3>
+      <input
```

Comment:
I'm not entirely sure about best practice, but I have the impression that inputs should always have a label, for accessibility reasons. Alternatetively, maybe an aria-label would work.